packages/nx-stylelint/src/executors/lint/executor.ts

```diff
@@ -70,4 +75,27 @@ export async function lintExecutor(
   };
 }
 
+function getFilesToFormat(format: string) {
+  return [...getUncommittedFiles(format), ... getUntrackedFiles(format)];
+}
+
+
+function getUncommittedFiles(format: string): string[] {
+  return parseGitOutput(`git diff --name-only --relative HEAD .`, format);
+}
+
+function getUntrackedFiles(format: string): string[] {
+  return parseGitOutput(`git ls-files --others --exclude-standard`, format);
+}
+
+function parseGitOutput(command: string, format: string): string[] {
```

Comment:
Hey @Phillip9587 ,

one question just realized lintFilePatterns is an array, to filter files I was thinking about using minimatch like below to get the files, is it ok to introduce minimatch, or should I look for other options.

`return format.map((pattern) => minimatch.match(files, pattern, { matchBase: true })).flat(); `

Reply:
Just use minimatch